website/src/components/index-page.tsx

```diff
@@ -497,8 +496,9 @@ export function IndexPage(): ReactElement {
         description={
           <div className="space-y-2">
             <p>
-              Thanks to <a href="https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API"></a> so
-              it can run on any <i>JavaScript</i> runtime.
+              Thanks to{' '}
+              <a href="https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API">Fetch</a>, it can
```

Comment:
```suggestion
              <a href="https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API">Fetch API</a>, it can
```

Reply:
@ardatan oops missed it. Adding it directly to master.